va/caa.go

```diff
@@ -52,9 +52,13 @@ func (va *ValidationAuthorityImpl) checkCAA(
 	if err != nil {
 		return probs.CAA("CAA records for %s were malformed", identifier.Value)
 	}
+	if challengeType == nil {
+		noneChallenge := "none"
```

Comment:
`unknown` might make more sense here.

Reply:
Also can you add a test for this in the table test below.

Reply:
wfm :point_right:  c9408e7

Reply:
"Also can you add a test for this in the table test below."

Oops, missed this comment. I'll push a commit for that as well.

Reply:
Test case added in 198bf23